CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
     -   Added logic for contolled state for the component if `selectedItems` and `onSelection` passed.
 -   `<StickyNoteModal/>`
     -   static test id `data-test-id="sticky-note-modal"` will be removed with next major version
+        `<MultiSuggestField />`
```

Comment:
Looks like the list item point is missing here, is it? So the auto-format does not work correctly.

Reply:
Yes, fixed the point, we had the same above - so I just added the current fix there